identity_service/dummy.env

```diff
@@ -13,3 +13,4 @@ BIOANALYZER_ENABLED=true
 BIOANALYZER_SERVICE_URL=http://bioanalyzer-service:8080
 BIOANALYZER_QUALITY_THRESHOLD=30
 REPLAY_ATTACK_ENABLED=true
+MAX_DIDS=100
```

Comment:
Do we need to set this in our remote envs before deploying?

Reply:
Yeah, we will need to do so... good catch.